src/searchbar/__tests__/SearchBar-ios.js

```diff
@@ -15,6 +15,13 @@ describe('iOS SearchBar component', () => {
     expect(toJson(component)).toMatchSnapshot();
   });
 
+  it('should render with visibleCancel prop', () => {
+    const component = shallow(<SearchBar theme={theme} visibleCancel />);
```

Comment:
Forgot to update the test.
```suggestion
    const component = shallow(<SearchBar theme={theme} showCancel />);
```